codex-rs/app-server-protocol/src/protocol/common.rs

```diff
@@ -280,6 +279,10 @@ client_request_definitions! {
         params: #[ts(type = "undefined")] #[serde(skip_serializing_if = "Option::is_none")] Option<()>,
         response: v1::GetUserAgentResponse,
     },
+    GetClientInfo {
+        params: #[ts(type = "undefined")] #[serde(skip_serializing_if = "Option::is_none")] Option<()>,
```

Comment:
@owenlin0 @pakrym-oai do we want to omit skip_serializing-if now? I do see it on the models around this though